server/src/main/kotlin/suwayomi/tachidesk/manga/impl/Manga.kt

```diff
@@ -75,7 +75,7 @@ object Manga {
 
             transaction {
                 MangaTable.update({ MangaTable.id eq mangaId }) {
-
+                    it[MangaTable.title] = fetchedManga.title
```

Comment:
I think we should guard against title changes before we implement this, downloads rely on the title

Reply:
JSON can change title of a manga: https://tachiyomi.org/help/guides/local-manga/#advanced

And we only get the title when the manga is first introduced into the database and there's no way to update it, the source might update the name and `fetchMangaDetails` can give the new name? I'm not sure how it works in Tachiyomi though...

Reply:
In main Tachi the json allows deciding the name of the manga, but it does not update it for future refreshes

Reply:
Basically no title changes are allowed to happen in main Tachi, most forks allow title changes because they properly rename the downloads folder.

Reply:
> most forks allow title changes because they properly rename the downloads folder.

We can do the same, rename the download dir when old title is not the same as the new title?

Reply:
Yep, that would work